src/components/Login/Login.css

```diff
@@ -0,0 +1,84 @@
+.overlay {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 100vw;
+    background: #d5fffe;
+    color: #343434;
+    position: absolute;
+    left: 0;
+    top: 0;
+}
+
+.wrapper {
+    display: flex;
+    flex-direction: column;
+    padding: 15px 24px;
+    max-width: 440px;
+    width: 60%;
+    border-radius: 7px;
+    box-shadow: 2px 5px 8px #ececec;
+    background: #fff;
+}
+
+.title {
+    font-size: 24px;
+    font-weight: 700;
+    margin-bottom: 15px;
+    /* letter-spacing: 18px; */
```

Comment:
Never commit commented out code. Always remove dead codes.

Reply:
addressed the comment.